data/remnant/remnant ships.txt

```diff
@@ -1108,8 +1108,14 @@ ship "Penguin"
 		"ion resistance" 0.05
 		"scramble resistance" 0.05
 		"thrust" 4.2
+		"thrusting energy" 0.65
+		"thrusting heat" 0.6
```

Comment:
```suggestion
		"thrusting energy" 0.31
		"thrusting heat" 0.29
```

As a built-in system, it dispenses with a lot of inefficiency that is required to support customization, modularity, etc. As such, the least efficient existing Remnant system (the Anvil) should be the maximum energy/heat cost per unit. There's no reason why such a system should be twice as inefficient as the anvil is.

Reply:
I got around 0.62 thrusting energy and 0.59 thrusting heat if using the anvil class engine efficiencies, so I've lowered the costs to 0.6 energy and 0.55 heat.